readme.md

```diff
@@ -1,14 +1,95 @@
+
+A popular stance among the Sauerbraten community is "that's impossible", and this is what we want to prove wrong. Our answer to remarks like "things are best like they are" is: standing still means falling behind.
+
+
+###### Different features are afoot giving a general but no inevitably defined idea of where we're going next:
+
+* node.js scripting
+ * (JavaScript + asynchronous + package manager) 
+* new user interface using HTML5 + CSS3
+* functionality to facilitate content-creating:
+  * better editing features (e.g. a version control system for maps)
+  * **extensibility** (e.g. for custom playermodels, sounds,..)
+* in-game content sharing
+* documentiation of the whole code base
+* refractored code to be more modern, scaleable, safe, non-blocking, modular, free, ... than before
+ * modularity (makes working as/in a team easier)
+ * scaleable and non-blocking resulting in better performance
+ * heavier use of proven third-party implementations of functionality instead of buggy and slow "reinvent-the-wheel" implementations
+* ...
+
+You may now find yourself struggling about one point or the other; nonetheless these points have already required a lot of thought and discussion, so chances are high that you will be convinced by either working with it, or by reflecting our thoughts.
+
+However, if you want to influence our development you should join our IRC-channel #inexor on gamesurge.net (see [Contact](https://github.com/inexor-game/code/wiki/Development-Environment)) and talk with us.  
+And of course we won't be too proud to give in if you face us with a more thought-through and better developed alternative.
+
+
+## What have we changed already?
+
+You can see our changelog [here](https://github.com/inexor-game/code/blob/master/changelog.md).
+
+Furthermore, the [Wiki](https://github.com/inexor-game/code/wiki) has more information.
+If you are a Sauerbraten developer worrying about us changing stuff and we're "totally going in the wrong direction", check out the posts in our blog called
+"Design Decisions" ([Design-Decisions Part1](https://inexor.org/blog/2015/04-26-alpha1-design-decisions), [Design-Decisions Part2](https://pad.inexor.org/p/Alpha3_Design_Decisions)) giving you a vague idea of our reasoning for some points.
+(You'll probably find more info on the dedicated wiki-sites, in the readme of the particular module and for specific stuff in the [documentation](http://docs.inexor.org) )
+
+
+A little note on that is that a lot of our decisions are consequences of other ones (e.g. working as a team had influences on the whole build structure, that we use cmake, that we use git and hence had to split the data and code repository and more ..  it's all chained).
+Consequently, our approach in developing Inexor is probably the most profound one in the Cube-world.
+
+
+# First Steps
+
+You have already accomplished the first step by reading this readme. Congratz!
```

Comment:
Add a "how to run"?

Clone ... into media/.
Click inexor_unix/the batch script.